src/Symfony/Component/BrowserKit/AbstractBrowser.php

```diff
@@ -204,7 +204,7 @@ public function getCookieJar(): CookieJar
      */
     public function getCrawler(): Crawler
     {
-        if (null === $this->crawler) {
+        if (!isset($this->crawler)) {
```

Comment:
Why this change here? That variable has a `protected` status in this class.

Reply:
This change anticipates a corresponding one on 7.0 when we'll add native types to public and protected properties.

Reply:
You could use `??` with a throw expression here.

```php
public function getCrawler(): Crawler
{
    return $this->crawler
        ?? throw new BadMethodCallException(sprintf('The "request()" method must be called before "%s()".', __METHOD__));
}
```